tests/backend/localmongodb/test_queries.py

```diff
@@ -180,3 +200,56 @@ def test_get_block():
 
     block = dict(query.get_block(conn, 3))
     assert block['height'] == 3
+
+
+def test_delete_unspent_outputs(db_context, utxoset):
+    from bigchaindb.backend import query
+    unspent_outputs, utxo_collection = utxoset
+    delete_res = query.delete_unspent_outputs(db_context.conn,
+                                              *unspent_outputs[::2])
+    assert delete_res['n'] == 2
+    assert utxo_collection.find(
+        {'$or': [
+            {'transaction_id': 'a', 'output_index': 0},
+            {'transaction_id': 'b', 'output_index': 0},
+        ]}
+    ).count() == 0
+    assert utxo_collection.find(
+            {'transaction_id': 'a', 'output_index': 1}).count() == 1
+
+
+def test_store_one_unspent_output(db_context, unspent_output_1):
+    from bigchaindb.backend import query
+    mongo_client = MongoClient(host=db_context.host, port=db_context.port)
+    utxo_collection = mongo_client[db_context.name].utxos
+    res = query.store_unspent_outputs(db_context.conn, unspent_output_1)
+    assert res.acknowledged
+    assert len(res.inserted_ids) == 1
+    assert utxo_collection.find(
+        {'transaction_id': unspent_output_1['transaction_id'],
+         'output_index': unspent_output_1['output_index']}
+    ).count() == 1
+
+
+def test_store_many_unspent_outputs(db_context, unspent_outputs):
+    from bigchaindb.backend import query
+    mongo_client = MongoClient(host=db_context.host, port=db_context.port)
+    utxo_collection = mongo_client[db_context.name].utxos
```

Comment:
This initialization is being repeated in multiple test cases, maybe this could be written in form of a fixutre or the `b` fixture could be updated to use `BigchainDB` class?

Reply:
addressed in f98af0bdd6d483eed5ffb814a25f0751efe36ed6